### DISCUSSION TITLE
Teacher Application Form Issues - Shockingly Bad Support

### DISCUSSION THREAD

**Discussion by marcel-goldschen-ohm** [community/community#50261] [GitHub Education] [answered]
I use GitHub extensively for both teaching (college level) and research. However, I have been unable to complete the form requesting verification of my teacher status at https://education.github.com/discount_requests/application. Every time I attempt to fill out the form I receive the error "To continue you must modify your browser settings to allow GitHub to use your location information." I have tried in both Safari and Firefox and have also set my locations permissions in both browsers to always allow location tracking for both https://github.com and https://education.github.com, but the error persists nonetheless. I tried opening a support ticket with GitHub (Ticket 2058476), but my ticket was closed without any explanation and only a single canned response message linking to generic instructions which are of no help. I understand that folks are busy, but to be honest I was quite shocked by this level of support as I expected better from GitHub. There are quite a few discussions where others also had problems with the teacher or student status verification (and also claimed to receive poor support), but in all those cases that I found the problems occurred after submitting the form whereas I cannot even submit the form in the first place. Any ideas on how to move forward to be verified as a teacher would be much appreciated.

Cheers,
Marcel

**Comment by samus-aran** [comment]
Hi @marcel-goldschen-ohm 

We're sorry to hear of your frustration. As we direct users to utilise the community more and share knowledge on all things GitHub Global Campus sometimes, like this, users require explicit help. But we're here to assist ⭐

Although I see you've tried using Firefox & Safari we've seen other discussions saying they've had success with Edge. Although not ideal it would help us perhaps identify there is a browser specific issue. Are you able to perform a hard refresh of your browser and include a screenshot of the location access being enabled whilst on the page? This would be helpful to present to the internal team for investigation.

**Reply by marcel-goldschen-ohm** [reply]
Hi samus-aran. I'll give edge a try, thanks for the suggestion. In the meantime I've included screenshots showing the enabled location access and the form error. I'm not sure what you mean by a hard refresh?
<img width="1100" alt="education github com location access" src="https://user-images.githubusercontent.com/7348010/227373251-c7920c60-3fb3-4618-b6d7-e52af6f2abcd.png">
<img width="833" alt="github com location access" src="https://user-images.githubusercontent.com/7348010/227373266-08726827-fde1-4a8f-bdc1-41fb33ceabbf.png">
<img width="864" alt="form error" src="https://user-images.githubusercontent.com/7348010/227373280-2981a0b2-0565-4bb4-a143-3f954443a852.png">

**Reply by marcel-goldschen-ohm** [reply]
Same error with Edge also after allowing location access.

**Comment by marcel-goldschen-ohm** [comment, marked as answer]
SOLVED!!! The issue was that my macOS operating system had location services disallowed despite them being enabled in Firefox. I guess this is due to my unfamiliarity with my OS rather than a browser issue. To help avoid this in the future you might want to add a note to also make sure your system allows location services in the error message shown in the image posted above. Or maybe I'm the only one who made this mistake ;) Cheers

**Reply by samus-aran** [reply]
That is awesome to hear @marcel-goldschen-ohm 🎉  That is a very excellent point and something we will include in our FAQs and documentation.
